assets/scss/components/_c-header-menu.scss

```diff
@@ -0,0 +1,131 @@
+/**
+ * This component defines the look of the desktop header menu.
+ * It's recommended to always use a negative value for the
+ * $sub-menu-offset value, to avoid mouse movement stuttering
+ * on the dropdowns.
+ */
+
+$sub-menu-offset: -10px;
```

Comment:
We could move the variables inside the selector, so the scope of that variable is closed to that selector rather than globally.